src/megatron/bridge/training/train.py

```diff
@@ -633,7 +636,7 @@ def train_step(
         )
 
         # Forward pass.
-        forward_backward_func = get_forward_backward_func()
+        # forward_backward_func = get_forward_backward_func()
```

Comment:
ack, this was a bad merge from https://github.com/NVIDIA-NeMo/Megatron-Bridge/pull/1058 that overrode this. both this line and this (https://github.com/NVIDIA-NeMo/Megatron-Bridge/blob/8fb6d0c3f882b539708630103e7dbe73c8e9f3ef/src/megatron/bridge/training/eval.py#L128) need to be removed